image/golem_blender_app/setup.py

```diff
@@ -26,7 +29,7 @@ def parse_requirements():
 
 setup(
     name='Golem-Blender-App',
-    version='0.1.0',
+    version=constants.VERSION,
```

Comment:
imo this value doesn't really matter anywhere and it's slightly hacky to reuse to image version here.

Reply:
I wouldn't call it 'reusing image version'. I just mean this value to be *the app version*. So it felt quite natural to use it both for Python package and Docker image.

Reply:
App version is the version of the whole app, i.e. dockerfile + this package. Why would changing the dockerfile affect the version of the python package?

Reply:
What would be a reason for these two to diverge?
Although if the package version is not used at all it could just stay `0.1.0` forever as well.

Reply:
The fact that this code is a package is just for the ease of management and installation. There's no meaning behind this versioning. I'm fine with anything here.